extensions/terminal-suggest/src/completions/upstream/git.ts

```diff
@@ -242,7 +241,7 @@ export const gitGenerators: Record<string, Fig.Generator> = {
 			"--no-color",
 			"--sort=-committerdate",
 		],
-		postProcess: postProcessBranches({ insertWithoutRemotes: true }),
+		postProcess: postProcessBranches(),
```

Comment:
These changes apply to `localRemoteBranches` and `localBranches`. there's another one `localOrRemoteBranches` which I think is what I'm after. I think before doing this we should understand why it's like this, and considering upstreaming a change if that's a possibility.

Reply:
moving to the next milestone as I think we'll need to ask the fig team why they did this (not show remotes by default)